Sources/Fullscreen/NotificationCenter/ActivityIndicatorSectionFooterView.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  ActivityIndicatorSectionFooterView.swift
+//  FinniversKit
+//
+//  Created by Granheim Brustad , Henrik on 04/02/2020.
+//  Copyright © 2020 FINN AS. All rights reserved.
```

Comment:
Shorten the header?

Reply:
Oh, yes. Forgot about that one

Reply:
I think we can standardize these nowadays, I'll add a task to look into that